cypress/integration/integration_tests/nav_test.js

```diff
@@ -6,6 +6,11 @@ describe('Integration test for the navbar', () => {
 
     // The page should now be switched to the Michael disaster.
```

Comment:
Do we want the URL-changing support at all, then?

Reply:
Removed